src/main/java/me/gaminglounge/estats/listener/GetDamage.java

```diff
@@ -25,22 +27,32 @@ public class GetDamage implements Listener {
 
 
     @EventHandler
-    public void onDamagedbBy(EntityDamageByEntityEvent event){
+    public void onDamagedBy(EntityDamageByEntityEvent event){
 
-        //damageTest(event.getDamage(), event.getEntity().getType(), event.getDamager().getName(),event.getDamageSource().getDamageType());
-        //damageDisplay(event.getDamage(),event.getDamager());
         EStats.INSTANCE.entityName.add_(event.getEntity());
-        if(event.getDamager() instanceof Player & event.getEntity() instanceof LivingEntity){
+        if((event.getDamager() instanceof Player)
+        || (event.getDamager() instanceof Projectile)
+        && (event.getEntity() instanceof LivingEntity)){
```

Comment:
Not what i ment!
You shuld encaspulate the or.
eg. `(e... || e...) && ...of Living...)`